### ISSUE TITLE
cursor.execute() rejecting variable passed to params argument

### ISSUE THREAD

**Issue by djrscally** [pymssql/pymssql#666]
I am attempting to fetch some data, passing in a tuple of parameter values to the `params` argument of `cursor.execute()`:

```Python
with pymssql.connect(server=server, port=port) as mssql:
    cursor = mssql.cursor(as_dict=True)
    cursor.execute(query.format(','.join('%d' for i in range(len(batch_keys)))), batch_keys)
    df = pd.DataFrame(cursor.fetchall())
```

This throws the following error:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-34-7e1e53cb45d9> in <module>
      2 with pymssql.connect(server='GBSHEWSQLPRD175', port='14330') as mssql:
      3     cursor = mssql.cursor(as_dict=True)
----> 4     cursor.execute(query.format(','.join('%d' for i in range(len(batch_keys[:5])))), batch_keys)
      5     df = pd.DataFrame(cursor.fetchall())

src\pymssql.pyx in pymssql.Cursor.execute()

src\_mssql.pyx in _mssql.MSSQLConnection.execute_query()

src\_mssql.pyx in _mssql.MSSQLConnection.execute_query()

src\_mssql.pyx in _mssql.MSSQLConnection.format_and_run_query()

src\_mssql.pyx in _mssql.MSSQLConnection.format_sql_command()

src\_mssql.pyx in _mssql._substitute_params()

src\_mssql.pyx in _mssql._quote_data()

src\_mssql.pyx in _mssql._quote_or_flatten()

ValueError: expected a simple type, a tuple or a list
```

The problem is that the value passed to the param argument **IS** a tuple. If I run `print(type(batch_keys), batch_keys)` the output is:

```
<class 'tuple'> (3079004, 2949733, 3444857, 3149177, 2897891)
```

And if I replace the reference to the batch_keys variable in the `.execute()` call with its actual output like so:

```Python
with pymssql.connect(server=server, port=port) as mssql:
    cursor = mssql.cursor(as_dict=True)
    cursor.execute(query.format(','.join('%d' for i in range(len(batch_keys)))), (3079004, 2949733, 3444857, 3149177, 2897891))
    df = pd.DataFrame(cursor.fetchall())
```

Then the whole thing works fine.

I have no idea what's going on there. Any ideas, please?

**Comment by djrscally** [comment]
tl;dr, Pandas doesn't actually support pymssql even though it works in a broad range of cases. Sorry to bother you all.
